static/js/components/VideoSubtitleCard.js

```diff
@@ -0,0 +1,63 @@
+// @flow
+import React from "react";
+
+import Card from "./material/Card";
+import { makeVideoSubtitleUrl } from "../lib/urls";
+
+import type { Video, VideoSubtitle } from "../flow/videoTypes";
+import Filefield from "./material/Filefield";
+import Button from "./material/Button";
+
+export default class VideoSubtitleCard extends React.Component {
+  props: {
+    video: Video,
+    isAdmin: boolean,
+    uploadVideoSubtitle: Function,
+    deleteVideoSubtitle: Function
+  };
+
+  render() {
+    const { video, isAdmin, uploadVideoSubtitle, deleteVideoSubtitle } = this.props;
+    return <Card className="videoSubtitle-card">
+      <div className="videoSubtitle-card-body">
+        <h4 className="mdc-typography--subheading2">
+          Subtitles
+        </h4>
+        <div className="actions">
+          <div>
+            {video.videosubtitle_set.map((subtitle: VideoSubtitle, key) => (
+              <div className="mdc-list-item" key={key}>
+                <span>{ subtitle.filename }</span>
+                <span>{ subtitle.language_name }</span>
+                <span>
+                  <a className="mdc-list-item mdc-link download-link"
+                    href={makeVideoSubtitleUrl(subtitle)} download={subtitle.filename} alt="Download this subtitle">
+                    <i className="material-icons">file_download</i>
+                  </a>
+                </span>
+                {isAdmin &&
```

Comment:
I think I prefer [this](https://github.com/mitodl/odl-video-service/pull/249/files#diff-4c959d8561c0601b9bdd92f6fb8af418R148) style of indenting when we render based on some boolean logic

Reply:
OK